src/codeflare_sdk/cluster/widgets.py

```diff
@@ -89,3 +100,234 @@ def is_notebook() -> bool:
         return True
     else:
         return False
+
+
+def view_clusters(namespace: str = None):
+    """view_clusters function will display existing clusters with their specs, and handle user interactions."""
```

Comment:
```suggestion
    """
    view_clusters function will display existing clusters with their specs, and handle user interactions.
    """
```
nit: Makes the file look more uniform if all of the head comments look like this :)